src/components/chip-set/chip-set.tsx

```diff
@@ -208,16 +196,19 @@ export class ChipSet {
     @State()
     private inputChipIndexSelected: number = null;
 
-    private mdcChipSet: MDCChipSet;
+    @State()
+    private selectedChip: Chip;
+
+    @State()
+    private selectedChipIds: Array<string | number>;
```

Comment:
So, can there only be one chip selected, or can there be multiple chips selected?

Reply:
> So, can there only be one chip selected, or can there be multiple chips selected?

I found the answer to this further down in the code.